config/config-es.json

```diff
@@ -494,6 +503,7 @@
         "buttonLogoutText": "Cerrar Sesión",
         "registeredVersionText": "Suscripción activa",
         "logoutAlertText": "¿Está seguro de que deseas cerrar la sesión?",
+        "titleUserIdText": "Identificación de usuario:",
```

Comment:
duplicate?

Reply:
I deleted the duplicate one in English..

Reply:
ES need to delete the one without , at end. 507